File: array.cpp
#include <iostream>
#include <cstdlib>
#include <algorithm>

using namespace std;

const int size=20;

int table[size];

int main(){
srand((unsigned)time(0));

//Print array:
cout << "The array contains: " <<endl;
for (int i=0; i <=20; i++){
    table[i] = ((rand() %100) + 0);

cout <<table[i]<<", ";

}
cout << '\n';
cout << "This array have "<<size<<" elements."<<endl;
int tableSize = sizeof(table)/sizeof(table[0]);

cout << endl<<"And it's sorted array now: ";
    sort(table,table+tableSize); //sort method from lowest to highest value
    for (int i=0; i<tableSize; i++)
    {
        cout << " "<< table[i]<<", " ;
    }
  return 0;

}

File: array.js
// Make the array with 20 elements.
var table = new Array(20);

// Confirm array's length.
console.log("The array have "+table.length+" elements.");

//Insert random integer numbers between 0 and 100
for (i=0; i<table.length;i++){
    table[i]=Math.floor(Math.random() * (100 - 0 + 1) + 0);
};

console.log("The array contains:"+table);
//Sort the table in alphabetical order
function compareNumbers(a,b){
    return a-b
};
table.sort(compareNumbers);

//Show contain of sorted array.
console.log("The array after sort operation:"+table);

